MarsRover/Map.cs

```diff
@@ -4,14 +4,10 @@ namespace MarsRover
 {
     public class Map
     {
-        public Map(int width, int height, List<Square> squares)
+        public Map(List<Square> squares)
         {
-            width = Width;
-            height = Height;
             Squares = squares;
         }
-        public int Width { get; private set; }
-        public int Height { get; private set; }

```

Comment:
This could still be useful information e.g. for when the rover is navigating. Just don't necessarily _need_ to include it as part of a file input.